infra/label_validation_agent_step_functions/lambdas/validate_labels.py

```
@@ -0,0 +1,858 @@
+            "llm_calls_successful": llm_calls_successful,
+            "llm_calls_failed": llm_calls_failed,
+            "rate_limit_errors": rate_limit_errors,
+            "server_errors": server_errors,
+            "retry_attempts": 0,  # Step Function handles retries, not Lambda
+            "circuit_breaker_triggers": 1 if circuit_breaker_triggered else 0,
+            "timeout_errors": timeout_errors,
+        },
+    }
+
+
+def handler(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
```

Ruff (0.14.6)

Unused function argument: `context`

(ARG001)